ons_alpha/jinja2/templates/base.html

```diff
@@ -10,6 +11,53 @@
     {% endif %}
 {% endblock meta %}
 
+
+{% block head %}
+    <link rel="stylesheet" href="{{ static('css/main.css') }}">
+    {% if GOOGLE_TAG_MANAGER_ID and not request.is_preview %}
+        <!-- Google Tag Manager -->
+        <script>
+            function loadGTM() {
+                (function (w, d, s, l, i) {
+                    w[l] = w[l] || [];
+                    w[l].push({ 'gtm.start': new Date().getTime(), event: 'gtm.js' });
+                    var f = d.getElementsByTagName(s)[0],
+                        j = d.createElement(s),
+                        dl = l != 'dataLayer' ? '&l=' + l : '';
+                    j.async = true;
+                    j.src =
+                        'https://www.googletagmanager.com/gtm.js?id=' +
+                    i +
+                    dl +
+                    '&gtm_preview=env-12&gtm_cookies_win=x';
+                    f.parentNode.insertBefore(j, f);
+                })(window, document, 'script', 'dataLayer', '{{ GOOGLE_TAG_MANAGER_ID|escapejs }}');
+            }
+
+            var a = /^(.*)?\s*'usage':true\s*[^;]+(.*)?$/;
+            if (document.cookie.match(a)) {
+                loadGTM();
+            }
```

Comment:
@zerolab @MebinAbraham  One for Beta... I think it would be great to define a set of helper methods in JS to help with this conditional stuff in future... e.g.

```
usage_cookies_are_permitted()
settings_cookies_are_permitted()
are_campaign_cookies_permitted(campaign_id)
```